hosts/web01/lemmy.nix

```diff
@@ -0,0 +1,47 @@
+{ inputs, pkgs, lib, ... }:
+
+let
+  domain = "lemmy.nix-community.org";
+in
+{
+  services.lemmy = {
+    enable = true;
+    nginx.enable = true;
+    database.createLocally = true;
+
+    settings = {
+      hostname = domain;
+    };
+  };
+
+  services.nginx.virtualHosts.${domain} = {
+    enableACME = true;
+    forceSSL = true;
+  };
+
+  # Lemmy image storage
+  services.pict-rs = {
+    enable = true;
+    dataDir = "/mnt/lemmy-pict-rs";
```

Comment:
It might be necessary to also extend the systemd unit to guarantee that the mount happens before the service gets started.

Reply:
`unitConfig.RequiresMountsFor = [ "" ];`

Reply:
I don't think it's required here as the mount point is systemd managed with autofs. I've added it anyway for good measure but I don't think it's strictly required.

Reply:
https://github.com/NixOS/nixpkgs/pull/243742